code/API_definitions/region-device-count.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.3
 info:
   title: Region Device Count
   description: |
-    This API allows for the API Consumer to Query the number of switch on devices in the specified area during a certain time window.The query area can be a circle or a polygon composed of longitude and latitude points.
+    This API allows for the API Consumer to query device count in the specified area during a certain time interval.The query area can be a circle or a polygon composed of longitude and latitude points.
   version: 0.1.1-wip
```

Comment:
Regarding the release management point ([here](https://wiki.camaraproject.org/display/CAM/API+versioning)) the version must be `wip`

Reply:
Thanks,the version has been updated